backends/openvino/quantizer/quantizer.py

```diff
@@ -119,15 +123,18 @@ def __init__(
             compression_mode = mode.value.replace(
                 "wo", ""
             )  # Mode value has to match NNCF CompressWeightsMode
-            weight_compression_configuration = get_weight_compression_configuration(
+            self._wc_config = get_weight_compression_configuration(
```

Comment:
Let's do `algo.get_config()` and sort outputs to keep only required params there, then we don't have to keep copies in `self._wc_config`